packages/scoped-custom-element-registry/src/scoped-custom-element-registry.js

```diff
@@ -88,7 +88,7 @@ if (!ShadowRoot.prototype.createElement) {
         this._awaitingUpgrade.delete(tagName);
         for (const element of awaiting) {
           pendingRegistryForElement.delete(element);
-          upgrade(element, definition);
```

Comment:
is there test coverage for this change, or were the tests already failing?

Reply:
Oh weird. I added a test... not sure what happened to it.

Reply:
Ok, added the test. PTAL.